app/app/src/main/java/io/mobjob/mentorship/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,80 @@
+package io.mobjob.mentorship.ui.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import io.mobjob.mentorship.R
+import io.mobjob.mentorship.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+
+    private var _binding: FragmentHomeBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        println("[SPCL_TAG] : " + "onCreateView")
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        goToFragmentDetails(Navigation.findNavController(view))
```

Comment:
navController is not needed here. You can delete it and use findNavController.navigate(R.id.action_detailsFragment_to_homeFragment) inside your method.